samples/AudioEcsSample/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+if(NOVELRT_VULKAN_SDK)
+  find_dependency(Vulkan REQUIRED)
+endif()
+
+set(SOURCES
+        main.cpp
+        )
+
+add_executable(AudioEcsSample ${SOURCES})
+add_dependencies(AudioEcsSample Resources)
+target_link_libraries(AudioEcsSample
+        PRIVATE
+        Engine
+        )
+target_compile_definitions(AudioEcsSample PUBLIC -DNOVELRT_MOLTENVK_VENDORED=$<IF:$<BOOL:${NOVELRT_MOLTENVK_VENDORED}>,1,0>)
```

Comment:
We really should open an issue about this and figure out a fix

Reply:
I'll open the issue but am concerned there won't be a true "fix" for this

Reply:
Opened #479 for tracking this